homeassistant/components/nfandroidtv/__init__.py

```diff
@@ -41,7 +37,6 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     try:
         await hass.async_add_executor_job(Notifications, host)
     except ConnectError as ex:
-        _LOGGER.warning("Failed to connect: %s", ex)
         raise ConfigEntryNotReady from ex
```

Comment:
The exception argument will be included in the default warning log for ConfigEntryNotReady.
```suggestion
        raise ConfigEntryNotReady("Failed to connect") from ex
```